src/Microsoft.PowerShell.ConsoleHost/resources/CommandLineParameterParserStrings.resx

```diff
@@ -186,6 +186,15 @@
   <data name="MissingConfigurationNameArgument" xml:space="preserve">
     <value>Cannot process the command because -Configuration requires an argument that is a remote endpoint configuration name. Specify this argument and try again.</value>
   </data>
+  <data name="MissingDebugPipeNameArgument" xml:space="preserve">
+    <value>Cannot process the command because -DebugPipeName requires an argument that is a name of the pipe you want to use. Specify this argument and try again.</value>
+  </data>
+  <data name="DebugPipeNameTooLongOnLinux" xml:space="preserve">
+    <value>Cannot process the command because -DebugPipeName specified is too long. Pipe paths on Linux can be up to 108 characters long. Your pipe path: {0}</value>
```

Comment:
> Pipe paths on Linux can be up to 108 characters long. Your pipe path: {0}

What the user specifies is the pipe name, not the pipe path, which would be `TEMP/CoreFxPipe_xxx...`. So (1) better say "pipe name" instead of "pipe path"; (2) is the 108 the length limit for the path or the name? better make sure we give the length limit for the name.

Reply:
cleaned up!
```
pwsh -debugpipename sdfhsardfghghjsdfdghkjhsdfghsdfghdfgssdfgsdfhkjsdhkjsdhksdfghk
Cannot process the command because -DebugPipeName specified is too long. Pipe names on this platform can be up to 55 characters long. Your pipe name 'sdfhsardfghghjsdfdghkjhsdfghsdfghdfgssdfgsdfhkjsdhkjsdhksdfghk' is 62 characters.
```